tests/E-MTAB-6870.condensed-sdrf.tsv

```diff
@@ -0,0 +1,180 @@
+E-MTAB-6870		ERR2619174	characteristic	cell line	Hep3B	http://www.ebi.ac.uk/efo/EFO_0002205
```

Comment:
there is another condensed SDRF file that is shorter in this same PR. Are we using both?

Reply:
Thanks for spotting this. I'm using example-conds-sdrf.tsv which is a trimmed version of the above file. Have removed it.